docs/conf.py

```diff
@@ -0,0 +1,58 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+sys.path.insert(0, os.path.abspath('..'))
+autodoc_mock_imports = ["sqlalchemy"]
+
+master_doc = 'index'
+
+# -- Project information -----------------------------------------------------
+
+project = 'sqlalchemy-ibmi'
+copyright = '2020, Kevin Adler and Naveen Ram'
+author = 'Kevin Adler and Naveen Ram'
+
+# The full version, including alpha/beta/rc tags
+release = '0.1.0'
+
+
+# -- General configuration ---------------------------------------------------
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+    "sphinx.ext.autodoc"
+]
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ['_templates']
+
+# List of patterns, relative to source directory, that match files and
+# directories to ignore when looking for source files.
+# This pattern also affects html_static_path and html_extra_path.
+exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store']
+
+
+# -- Options for HTML output -------------------------------------------------
+
+# The theme to use for HTML and HTML Help pages.  See the documentation for
+# a list of builtin themes.
+#
+html_theme = 'alabaster'
```

Comment:
```suggestion
html_theme = 'alabaster'
```

I'd prefer the read the docs theme. See https://github.com/IBM/python-itoolkit/blob/master/docs/conf.py#L90

Reply:
That also requires https://pypi.org/project/sphinx-rtd-theme/ to be installed. This should be added as a dev dependency (along with sphinx and the other packages)